src/Serilog.Sinks.AzureDataExplorer.Tests/FileSetTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿
+using System.Text;
+using FluentAssertions;
+using Serilog.Sinks.AzureDataExplorer.Durable;
+using Serilog.Sinks.AzureDataExplorer.Extensions;
+
+namespace Serilog.Sinks.AzureDataExplorer.Tests;
+
+public class FileSetTests : IDisposable
+{
+    private readonly string m_fileNameBase;
+    private readonly string m_tempFileFullPathTemplate;
```

Comment:
Nitpick , consistent naming. Some classes member variables do not have the m_ prefix.